src/main/java/uk/gov/digital/ho/hocs/audit/application/configuration/GsonConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.digital.ho.hocs.audit.application.configuration;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParseException;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.lang.reflect.Type;
+import java.time.LocalDateTime;
+
+@Configuration
+public class GsonConfiguration {
+
+    @Bean
+    public Gson gson() {
+        GsonBuilder gsonBuilder = new GsonBuilder();
+        gsonBuilder.registerTypeAdapter(LocalDateTime.class, new LocalDateTimeDeserializer());
+        return gsonBuilder.create();
+    }
+
+    static class LocalDateTimeDeserializer implements JsonDeserializer<LocalDateTime> {
```

Comment:
As above, how do we know we still need this, these were added in the early days of JSR-310